packages/playground/src/pages/test-issue7.astro

```diff
@@ -0,0 +1,22 @@
+---
+import { Markdown } from "astro-remote";
+
+// Working as expected
+const c0 = `Most .`;
+
+// 1 space
+const c1 = `Most \\
+.`;
+
+// 2 spaces
+const c2 = `Most  \\
+.`;
+
+// Space at the end
+const c3 = `Most . `;
```

Comment:
Just simplifying
```suggestion
const c0 = "Most .";
const c1 = "Most \n.";
const c2 = "Most  \n.";
const c3 = "Most . ";
```

Reply:
yeah, the only reason it was following the specific formatting was due to the OP of issue #7 referring to that specific formatting. 😅

Reply:
Right, but it's slightly concerning that `\\` is needed here? This is technically closer to OP's format. 🤷

Reply:
Co-Pilot explained it pretty good, but its mainly just weird MD BS... both ways in this case are correct, but the general preference is definitely the way your writing it over the OG format presented to us. MD does weird stuff with spaces...